src/styles/blocks/catalog.scss

```diff
@@ -17,9 +18,16 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: transform $color-title-smoothly, color $color-title-smoothly;
+
+    &:hover {
+      color: #34568b;
+      transform: scale(120%);
```

Comment:
Use '1.2' instead of '120%' for the transform scale value to increase the card size by 20% on hover